topics/reference_guide/work_with_icons_and_images.md

```diff
@@ -214,17 +214,17 @@ Register resource bundle via `com.intellij.iconDescriptionBundle` extension poin
 Create `icon.<icon-path>.tooltip` key in given resource bundle, where `<icon-path>` is the icon path with leading slash and `.svg` removed and slashes replaced with dots
 (e.g., `/nodes/class.svg` &rarr; `icon.nodes.class.tooltip`).
 
-## Mapping New UI Icons
+## New UI Icons
 
 > This feature is available since 2022.3.
 
-To fully support [New UI](https://www.jetbrains.com/help/idea/new-ui.html), the plugin must provide additional dedicated icons and mapping information.
-This allows supporting both UI variants at the same time — whichever the user chooses to use.
+To fully support the [New UI](https://www.jetbrains.com/help/idea/new-ui.html), the plugin must provide additional dedicated icons and mapping information.
```

Comment:
Is "the" needed here? It is skipped in the sentences below.

Reply:
I commented in the PR that I like to change the procedure list below as well because we left all articles out there.

"We saw New York" and "We saw _the_ New UI". IMO, we cannot just declare "New UI" as a [proper noun](https://en.wikipedia.org/wiki/Proper_noun) and leave the article out. We capitalize it to make it stand out, but it hurts when I read it without an article. Here are some examples from that Wiki page:

- The 16th robotic probe to land on the planet was assigned to study the north pole, and the 17th probe the south pole.
(common-noun senses throughout)
- When Probe 17 overflew the South Pole, it passed directly over the place where Captain Scott's expedition ended.
(in this sentence, Probe 17 is the proper name of a vessel, and South Pole is a proper name referring to Earth's south pole)
- Sanjay lives on the beach road.
(the road that runs along the beach)
- Sanjay lives on Beach Road.
(as a proper name, Beach Road may have nothing to do with the beach; it may be any distance from the waterfront)

Reply:
Oh, I'm sorry.

I noticed the screenshot, but my brain probably took it as a part of the applied changes diff and thought there were more.